source/plugins/hestia/dimmableLight/DimmableLight.py

```
@@ -17,3 +17,7 @@ def name(self):
     def pluginType(self):
         return "Light"
 
+    @staticmethod
+    def getEmptyRequiredInfo():
+        return {"ip": "127.0.0.1", "port": "0"}
+
```

Here you have default values, so the method name is a bit misleading

---

I think getDefaultRequiredInfo would indeed be better I will change that and then merge it